cmd/oras/internal/display/status/progress/status.go

```diff
@@ -35,6 +35,12 @@ const (
 	zeroDigest   = "  └─ loading digest..."
 )
 
+var (
+	spinnerColor  = aec.LightYellowF
+	doneMarkColor = aec.LightGreenF
+	progressColor = aec.LightBlueB
```

Comment:
Is the `LightBlueB` visible in Windows PowerShell?

Reply:
PS by default uses dark blue I think

![image](https://github.com/oras-project/oras/assets/4254263/d36e2e8a-66ac-42fa-84bf-d200aa19455f)

Reply:
From my view, the bar can be seen clearly
![image](https://github.com/oras-project/oras/assets/4254263/71d199f9-afbb-4879-a99e-521cb2543b42)